docs/shared/migration/adding-to-existing-project.md

```diff
@@ -322,21 +302,30 @@ To connect to Nx Cloud:
 - Commit and push your changes
 - Go to [https://cloud.nx.app](https://cloud.nx.app), create an account, and connect your repository
 
-#### Connect to Nx Cloud Manually
-
-If you are not able to connect via the automated process at [https://cloud.nx.app](https://cloud.nx.app), you can connect your workspace manually by running:
-
 ```shell
 npx nx connect
 ```
 
-You will then need to merge your changes and connect to your workspace on [https://cloud.nx.app](https://cloud.nx.app).
+When you are prompted to choose your CI provider, choose `GitHub Actions`.
+
+Once the script is finished, it will print a link in the terminal to register your repository in your [Nx Cloud](https://cloud.nx.app) account. Click the link and follow the steps provided to make sure Nx Cloud is enabled on the main branch of your repository.
```

Comment:
The browser window should open automatically, and users need to follow the steps until Nx Cloud is enabled on main branch (i.e. token is added). I think having them check `nxCloudAccessToken` to verify is important before continuing to next steps. User also need to `git pull origin main` before continuing.

(same for other standalone tutorials)

Reply:
The browser window does not open automatically